### ISSUE TITLE
Feature request: Show properties for object

### ISSUE THREAD

**Issue by fflaten** [TylerLeonhardt/vscode-inline-values-powershell#3]
Awesome extension Tyler! 👏

Would it be possible to show properties for objects? 

In the vscode api proposal issue they showed inline demo like `varname = object { prop: value, anotherProp : array[2] }`. Maybe we could show the default string-output format for pscustomobjects (ex output of `"$((dir)[0] | select *)"`)

Or make inline-value show the referenced property-value. Currently shows variable "var" even if line only uses `$var.SomeProp`

**Comment by TylerLeonhardt** [comment]
This is a hard one because although `varname = object { prop: value, anotherProp : array[2] }` looks nice, one with 50 properties might not look as nice.

@weinand what do you think about this? I imagine there is a generic ask here since it would apply to any inline value variable lookup usages.

**Comment by fflaten** [comment]
I agree with the length-issue for big objects. Just mentioned the `@{Param=Value1;Param2=Value2}` style since an equivalent in js was part of the [api issue](https://github.com/microsoft/vscode/issues/105690#issuecomment-758671619). Would probably get truncated too often to be useful. 

That's why the second proposal was added since that might be more realistic. Maybe possible using `InlineValueExpression` if accessing a  property is detected?

**Comment by fflaten** [comment]
Expression works for properties. Tested with a hardcoded pattern as PoC.

```
// Demo expression for property-view
const rng = new vscode.Range(6, 10, 6, 28);
allValues.push(new vscode.InlineValueEvaluatableExpression(rng, '$files[0].FullName'));
```

![image](https://user-images.githubusercontent.com/3436158/114617005-dc094a80-9ca7-11eb-9a23-ec5bd9505690.png)

**Comment by TylerLeonhardt** [comment]
Interesting... I guess we could support properties. That would be pretty nice!
